### ISSUE TITLE
Does not work with react-native-vision-camera v3

### ISSUE THREAD

**Issue by jacquesdev** [mrousavy/vision-camera-image-labeller#23]
Hello @mrousavy 

This plugin doesn't seem to work on v3 on either Android or iOS.

Are you able to take a look at what it would entail? I realise you may need sponsorship to do that, so please let me know the amount of effort and cost involved for anyone else interested in getting it working

**Comment by mrousavy** [comment]
Yea I can upgrade it to V3, I think that'd be 5 hours of effort - I'd do it for $400 :)

**Comment by ealmiladi** [comment]
> Yea I can upgrade it to V3, I think that'd be 5 hours of effort - I'd do it for $400 :)

Why is this plugin still up and referenced on the Vision Camera docs if it doesn't work on v3? Either support the latest or be very explicit that it only works on < 3.

**Comment by mrousavy** [comment]
@ealmiladi you're right. Here you go: https://github.com/mrousavy/vision-camera-image-labeler/commit/33de52dbc2391e215a1df04f8993791783070b60

**Comment by ste00martin** [comment]
@mrousavy I'm open to sponsoring it - would it work for just v3 or the new v4 you've recently released?

**Comment by mrousavy** [comment]
V4 and V3 are almost identical, only an import has changed. I can upgrade to make it work on V4 if you want :)

**Comment by fvonhoven** [comment]
> Yea I can upgrade it to V3, I think that'd be 5 hours of effort - I'd do it for $400 :)

Sounds good, what's your Venmo? This thing is dead in the water otherwise ;)

**Comment by lironsfadia** [comment]
This appears as example in the official docs: https://react-native-vision-camera.com/docs/guides/frame-processors#using-community-plugins, but still not working :(

**Comment by CMLCNL** [comment]
@mrousavy i can pay 400 usd just do it please. we want to support vision camera. i can send money with usdt

**Comment by aiden-petersen** [comment]
Hey @CMLCNL I'd be happy to do it for $400, let me know
